### PR TITLE
Add logging for the computationally expensive parts.

### PR DESCRIPTION
**Pull request by marianamarasoiu**
Addresses issue #2

### CODE REVIEW

File: lib/src/visualisation.dart [hunk 1]
```diff
@@ -13,13 +12,23 @@ class Visualisation {
   Range yRange;
   Range valueRange;
   bool transformsApplied;
+  Logger logger;
 
-  Visualisation([String outputFolder = "output/image/", String outputFilename]) {
+  Visualisation([String outputFolder = "output/image/", String outputFilename,
+      bool logging = true]) {
```
Comment (lukechurch) [inline lib/src/visualisation.dart]: enableLogging?

File: lib/src/visualisation.dart [hunk 2]
```diff
@@ -111,29 +134,39 @@ class Visualisation {
...
+    logger.info('Pixel values set. Duration/ms: ${sw.elapsedMilliseconds}');
+    sw.reset();
 
     String autoFileName = "${new DateTime.now()}"
-      "_${areaX.min},${areaX.max}_${areaY.min},${areaY.max}.png";
+        "_${areaX.min},${areaX.max}_${areaY.min},${areaY.max}.png";
     String filePath = _computeFilePath(autoFileName);
-   _writeImageSync(filePath, image);
+    _writeImageSync(filePath, image);
+    logger
```
Comment (lukechurch) [inline lib/src/visualisation.dart:168]: More conventional to break at the edge of the argument not the method selector

**Comment by lukechurch** [comment]
lgtm with a couple of nits

File: lib/src/visualisation.dart (hunk 2 above)
Comment (marianamarasoiu) [inline lib/src/visualisation.dart:168]: Dart Style did it :-)

**Comment by marianamarasoiu** [comment]
Thanks! Merging.

**Pull request merged**